nextcloud/README.md

```diff
@@ -58,7 +59,11 @@ Other tags than `daily` are built weekly. For security reasons, you should occas
 - **DB_USER** : username for database *(default : none)*
 - **DB_PASSWORD** : password for database user *(default : none)*
 - **DB_HOST** : database host *(default : none)*
-
+- **DATASTORE_BUCKET** : S3 bucket to use for primary storage *(default : none)*
+- **DATASTORE_KEY** : S3 Key for about bucket  *(default : none)*
+- **DATASTORE_SECRET** :  S3 secret for about bucket *(default : none)*
+- **DATASTORE_HOST** : S3 host for above bucket  *(default : none)*
+- **DATASTORE_PORT** :  S3 port for above bucket *(default : 443)*
 Don't forget to use a **strong password** for the admin account!
```

Comment:
You might want to add `DATASTORE_USE_PATH_STYLE` with default as `false`, since AWS buckets currently use path style. Path style tends to be for very specific 3rd party implementations of S3.